specification/indices/reload_search_analyzers/ReloadSearchAnalyzersRequest.ts

```diff
@@ -55,5 +55,9 @@ export interface Request extends RequestBase {
     allow_no_indices?: boolean
     expand_wildcards?: ExpandWildcards
     ignore_unavailable?: boolean
+    /**
+     * Changed resource to reload analyzers from if applicable
+     */
+    resource?: string
```

Comment:
This parameter is read in https://github.com/elastic/elasticsearch/blob/d9e751602dc4a874ef381797844f80044889fbef/server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestReloadAnalyzersAction.java#L42 since [8.10.0](https://github.com/elastic/elasticsearch/pull/96986).